docs/proposals/scheduling-suspension/README.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Support for resource scheduling suspend and resume capabilities
+
+authors:
+- "@Vacant2333"
+- "@Monokaix"
+
+reviewers:
+- "@RainbowMango"
+- "@XiShanYongYe-Chang"
+- "@a7i"
+
+approvers:
+- "@kevin-wangzefeng"
+- "@RainbowMango"
+- "@XiShanYongYe-Chang"
+creation-date: 2024-10-13
+
+---
+
+# Support for resource scheduling suspend and resume capabilities
+
+## Summary
+
+<!--
+Karmada 目前已经允许用户通过 Suspension 来暂停和恢复资源的 propagation 行为，这提高了 Karmada 的灵活性。
+
+但是目前 Suspension 支持的不够彻底，在 Karmada 中资源的分发可以简单理解为两个阶段，资源的调度和资源的传播。
+
+目前传播阶段可以较为灵活的由用户暂停和恢复，但是没有在调度阶段，没有提供暂停和恢复的能力。
+
+我们希望在 Karmada 的基础之上，通过暂停调度的方式，实现队列和配额管理等高阶能力，从而将集群有限的资源优先调度给高优先级的工作负载，增强 Karmada 在多集群调度的能力，从而满足用户在复杂场景下的需求。
+
+本文基于 [#5118](https://github.com/karmada-io/karmada/pull/5118) 在其中的 Suspension 基础之上，提供了暂停资源调度的能力。
+-->
```

Comment:
Please remove all Chinese content from the proposal. We should only use English to describe and review the proposal to avoid issues like being out of sync.